api/package.json

```diff
@@ -6,35 +6,49 @@
   "scripts": {
     "start": "nodemon server.js",
     "tunnel": "dotenv -- cross-var npx localtunnel --port 3000 --subdomain %API_TUNNEL_SUBDOMAIN%",
+    "lint": "eslint .",
     "test": "jest"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@bugsnag/js": "^7.20.2",
-    "@bugsnag/plugin-express": "^7.19.0",
     "airtable": "^0.11.1",
     "axios": "^0.25.0",
     "cors": "^2.8.5",
     "dayjs": "^1.11.1",
     "dotenv": "^10.0.0",
+    "eslint": "^8.46.0",
     "express": "^4.17.1",
     "i": "^0.3.7",
+    "lint-staged": "^13.2.3",
     "node-cron": "^3.0.0",
     "node-fetch": "^3.0.0",
     "nodemon": "^2.0.12",
+    "prettier": "^3.0.0",
     "vimeo": "^2.1.1"
   },
   "devDependencies": {
     "@faker-js/faker": "^8.0.2",
     "babel-cli": "^6.26.0",
     "babel-preset-env": "^1.7.0",
+    "@bugsnag/js": "^7.20.2",
```

Comment:
Bugsnag is actually a runtime dependency, so does need to be in `dependencies` - `devDependencies` is essentially for tooling that is only used for local development or testing.